ipmiMgrApp/Db/power_mngr.template

```diff
@@ -0,0 +1,6 @@
+record(ai, "$(P)$(R)$(id)Pwr") {
+  field(DESC, "Steady state power draw")
+  field(SCAN, "10 second")
+  field(INP, "$(P)$(R)PM2CurrCh$(power_channel)-Mon")
+  field(EGU, "W")
+}
```

Comment:
Why couldn't this use an alias?

Also, if you want, instead of having a scan, you can simply omit the `SCAN` field and use a `{your_pv} CP` link and it will be processed everytime the power draw changes.